packages/components/src/checkbox-control/stories/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { text } from '@storybook/addon-knobs';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import CheckboxControl from '../';
+
+export default { title: 'Checkbox Control', component: CheckboxControl };
+
+export const _default = () => {
+	const [ isChecked, setChecked ] = useState( true );
+	return (
+		<CheckboxControl
+			label="Is author"
+			checked={ isChecked }
+			onChange={ setChecked }
+		/>
+	);
+};
+
+export const All = () => {
```

Comment:
Linter doesn't allow to use `all` as the name of `const` because of React rules with hooks.

Reply:
I think the idea was to differentiate between stories and components this way. At least that's how the Storybook authors wrote the examples.